Update issue templates (#105)
kommitters/tickspot.js: .github/ISSUE_TEMPLATE/epic_feature.md

---

> name: 🎯 Epic feature
> about: Tell us about this new epic feature
> title: ''
> labels: '🎯 Epic, 👍 Feature'
> assignees: ''

Reviewer: <!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏An Epic is not always a feature, we could have an Epic for a list of fixes or refactors. So, what about just keeping "Epic" instead of "Epic feature"?

```suggestion
name: 🎯 Epic
about: Tell us about this new Epic
title: ''
labels: '🎯 Epic'
assignees: ''
```

Author: Ok, I hadn't thought that an Epic could be something other than a feature.
I will change it. :+1:

---

:information_source: This issue is an Epic :dart: 

## Context

Provides a context of the new feature and tells us, how it should be broken down into smaller implementations.

## Objective

Mention the objective of this new implementation.

## Checklist
<!--

> This checklist can be taken as the subtasks to solve this epic feature.
> If you consider, you can mention a related issue.

Reviewer: <!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏Maybe this comment is more understandable, do you agree?

```suggestion
This checklist can be taken as the subtasks to solve this Epic.
Whenever possible, mention the related issues.
```
